bot/skills/timer.py

```diff
@@ -0,0 +1,39 @@
+import logging
+from telegram import Update
+from telegram.error import BadRequest
+from telegram.ext import Updater, CommandHandler, CallbackContext
+from typing import List
+from time import sleep
+
+logger = logging.getLogger(__name__)
+
+def add_timer(upd: Updater, handlers_group: int):
+	logger.info("registering timer handler")
+	dp = upd.dispatcher
+	dp.add_handler(CommandHandler("timer", timer, run_async=True), handlers_group)
+
+
+def timer(update: Update, context: CallbackContext):
+	logger.info("making timer")
+
+	minutes = 0
+
+	try:
+		minutes = int(context.args[0])
+	except:
+		context.bot.send_message(
+      		update.effective_chat.id,
+        	"Не пиши глупости, укажи нормальное значение для таймера"
+        )
+
+		return
+
+	SEC_IN_MINUTE = 60
```

Comment:
Variables written in the format of screaming snake case must be declared before all functions and procedures and after all imports